.rustfmt.toml

```diff
@@ -0,0 +1,2 @@
+use_small_heuristics = "Max"
```

Comment:
This file and the formatting updates to already-merged code should go in their own PR, to separate formatting changes from additions.

Reply:
Ok, I'm making #5